src/GUI/UI/CMakeLists.txt

```diff
@@ -36,6 +37,7 @@ set(GUI_UI_HEADERS
         UIHouseEnums.h
         UIHouses.h
         UIInventory.h
+        UIItemOffset.h
```

Comment:
Also some thoughts:
Maybe we will be able to move some more code related to item grid processing out of inventory/chest processing.
So maybe it will be better to name this file ItemFunctions.cpp/h?

Reply:
GUI/UI/ItemGrid.h/cpp sounds good